src/model.jl

```diff
@@ -449,6 +466,14 @@ function get_name(aux::AuxRef, include_subindex=false)
     end 
 end
 
+function get_name(im::ImplicitvarRef, include_subindex=false)
+    # if im.subindex===nothing || include_subindex===false
+        return im.model._implicitvars[im.index].name
+    # else
+    #     return Symbol("$(im.model._implicitvars[im.index].name )[$(join(string.(im.subindex_names), ", "))]") 
+    # end 
+end
```

Comment:
Again, I left the lines in re indexed versions, but I don't think that is a thing.